pkg/apis/pingcap/v1alpha1/pd_config.go

```diff
@@ -179,9 +178,14 @@ type PDReplicationConfig struct {
 	// +optional
 	LocationLabels StringSlice `toml:"location-labels,omitempty" json:"location-labels,omitempty"`
 	// StrictlyMatchLabel strictly checks if the label of TiKV is matched with LocaltionLabels.
-	// Immutable, change should be made through pd-ctl after cluster creation
+	// Immutable, change should be made through pd-ctl after cluster creation.
+	// Imported from v3.1.0
```

Comment:
```suggestion
	// Available from PD v3.1.0.
```

Reply:
I think this should be as same as the tidb configuration